x-pack/legacy/plugins/apm/public/components/app/Settings/SettingsList.tsx

```diff
@@ -173,6 +173,33 @@ export function SettingsList() {
     />
   );
 
+  const failurePrompt = (
+    <EuiEmptyPrompt
+      iconType="alert"
+      title={
+        <h2>
+          {i18n.translate(
+            'xpack.apm.settings.agentConf.configTable.failurePromptTitle',
+            { defaultMessage: 'Failed to initialize settings.' }
+          )}
+        </h2>
+      }
+      body={
+        <>
+          <p>
+            {i18n.translate(
+              'xpack.apm.settings.agentConf.configTable.failurePromptText',
+              {
+                defaultMessage:
+                  'Whoops! The settings management failed to initialize. This might be because the user that is logged in does not have permission to read or write the APM agent configuration index. The next time a user with adequate privileges loads this screen, Kibana will complete this step.'
```

Comment:
This text is still confusing. Our implementation details wrt creating the index lazily should not shine through to the end-user. 

If we assume the apm_user has read/write access to the agent configuration index we have a couple of use cases:
 - An unknown backend error occurred (network glitch, database corruption, anything). We do not handle this in the rest of APM UI except for showing the automatic toast that the request failed. Feel free to explicitly handle the failure state. In that text the text needs to be something vague like "The list of agent configurations could not be fetched".
 - The user has neither read or write permission: they'd get the above error. That's better than most other places in APM UI.
 - The user has read permission but not write permission: the user should be able to see the list (and shouldn't see errors regardless if the index could not be created)
 - The user has read and write permissions: the user should be able to see the list